source/plugins/ruby/kubernetes_container_inventory.rb

```diff
@@ -165,6 +187,7 @@ def getContainersInfoMap(podItem, isWindows)
             podContainers.each do |container|
               containerInfoMap = {}
               containerName = container["name"]
+              containerInfoMap["image"] = container["image"]
```

Comment:
why is this added as part of this change ?

Reply:
> why is this added as part of this change ?

> This is getting image from "spec" .